pages/dashboard/index.tsx

```diff
@@ -7,15 +7,81 @@ import nextI18nextConfig from 'next-i18next.config';
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
 import { GetStaticProps } from 'next';
 import withAuth from '@/components/HOCs/withAuth';
+import Transaction from './Transaction';
+import { FiUser, FiFileText, FiDollarSign } from 'react-icons/fi';
+import { Chart as ChartJS, CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip, Legend } from 'chart.js';
+import { Line } from 'react-chartjs-2';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip, Legend);
 
 type Props = {
   loading: boolean;
 };
+enum CheckReduceIncrease {
```

Comment:
Duplicated declaring. It was declared in Transaction. Please declare it in @types